mmv1/templates/terraform/post_create/privateca_authority_enable.go.erb

```diff
@@ -0,0 +1,11 @@
+	url, err = replaceVars(d, config, "{{PrivatecaBasePath}}projects/{{project}}/locations/{{location}}/caPools/{{pool}}/certificateAuthorities/{{certificate_authority_id}}:enable")
+	if err != nil {
+		return err
+	}
+
+	log.Printf("[DEBUG] Enabling CertificateAuthority: %#v", obj)
+
+	res, err = sendRequest(config, "POST", billingProject, url, userAgent, nil)
```

Comment:
this returns a LongRunningOperation object and doesn't always finish immediately. Do you need to await these operations anywhere before doing something else?